manifests/runner.pp

```diff
@@ -132,7 +132,9 @@
     $ssh_password = undef,
     $require = [ Class['gitlab_ci_multi_runner'] ],
 ) {
-    $description = $name
+    # Single Quote Description so that none of the commands that use
+    # it break on spaces
+    $description = "'${name}'"
```

Comment:
My only concern here is that I was unaware spaces were even allowed - now I'm worried about single quotes/apostrophes breaking this fix.